package.json

```diff
@@ -10,7 +10,7 @@
     "node": ">=10.1.0"
   },
   "scripts": {
-    "start": "webpack-dev-server --mode development --open",
+    "start": "webpack-dev-server --mode development --open --port 3000",
```

Comment:
Should this be changed? I thought right now this change is for localhost only.

Reply:
Good point, I don't think this should be deployed.

Reply:
Removed from PR